composer.json

```diff
@@ -16,6 +16,7 @@
         "equip/structure": "^1.0",
         "filp/whoops": "^2.0",
         "lukasoppermann/http-status": "^2",
+        "monolog/monolog": "^1.19",
```

Comment:
@shadowhand can you think of a better way to log exceptions in `ExceptionHandler` without requiring an implementation of `LoggerInterface`?
```
try {
     $this->injector->make(LoggerInterface::class)->error($e->getMessage(), [
         'exception' => $e
     ]);
} catch (InjectionException $e){

}
```